### DISCUSSION TITLE
Restrict onnxruntime to use a specific opset at max?

### DISCUSSION THREAD

**Discussion by ai2ys** [microsoft/onnxruntime#13438] [Other Q&A] [answered]
Hi, 

I am quantizing a CNN model using onnxruntime. The target hardware supports only ONNX IR version 7 and Opset 13. Is it possible to restrict onnxruntime to use at maximum a specific ONNX IR version and Opset? 

Or is the ONNX IR and the Opset applied, that have been used when creating the ONNX file?

Thanks,
Sylvia

**Comment by faxu** [comment, marked as answer]
You should be able to specify opset version during model conversion.
